named-tunnel-k8s/cloudflared.yaml

```diff
@@ -31,6 +31,7 @@ spec:
             path: /ready
             port: 2000
           failureThreshold: 1
+          initialDelaySeconds: 10
```

Comment:
This is a start, but I'm not sure 10sec is enough. Under normal circumstances the edge propagation seems to be up to 20sec, so I'd be happier with 30 here.